src/components/Details/Details.js

```diff
@@ -64,34 +65,38 @@ class Details extends React.Component {
                  backgroundSize: 'cover'
             }
 
+            let details = <section className='details-content'>
```

Comment:
The error handling looks good.My only question is if it would make more sense to now push this information into a separate functional component now that Details is a class and has more code? This component is starting to get bulky.